### DISCUSSION TITLE
1.6.1

### DISCUSSION THREAD

**Discussion by kirb** [LegacyUpdate/LegacyUpdate#102] [Announcements]
* Fixes Windows Vista/7 Service Packs not installing due to a missing .exe extension on the file Legacy Update downloads.
* Legacy Update setup now requests Windows to not put the PC to sleep while it's running.

<hr /><em>This discussion was created from the release <a href='https://github.com/kirb/LegacyUpdate/releases/tag/v1.6.1.0'>1.6.1</a>.</em>

**Comment by ghost** [comment]
Sadly, it refuses to load from the link you provided. I think I have a 2 part problem: 1. Vista hates most modern websites, IE: Github. (No issues on legacydownload.net) 2.the download needs to happen from the machine that will use it (No-Brainer there!)

**Comment by ghost** [comment]
PS: Is the mew one up on the legacy site yet?

**Reply by kirb** [reply]
It should be, might need to press Ctrl-F5 to force a reload. If not, you can go directly to this link: http://content.legacyupdate.net/legacyupdate/LegacyUpdate-1.6.1.exe

**Comment by Santi551100** [comment]
Why I'm failed to install Error code: 1058

**Reply by kirb** [reply]
Is this on 1.6.1? Could you try with 1.6.2 and let me know if it still happens?

**Reply by Santi551100** [reply]
Ummm now is Error code: 3017

**Reply by Santi551100** [reply]
![Uploading 20230128_121412.jpg…]()

**Reply by kirb** [reply]
That error is "The requested operation failed. A system reboot is required to roll back changes made." - what was the last thing it was trying to install?

P.S. your screenshot failed to upload - could you try again? You need to wait for it to finish uploading before posting.
